src/types/gecko-profile.js

```diff
@@ -115,6 +125,12 @@ export type GeckoSampleStructWithResponsiveness = {|
   stack: Array<null | IndexIntoGeckoStackTable>,
```

Comment:
Not for this patch: I feel like these 2 types `GeckoSampleStructWithResponsiveness` and `GeckoSampleStructWithEventDelay` are used only in `process-profile.js`, because they're temporary structures used only during the processing. Therefore I feel strange and confusing that they're in this type file. But maybe that's useful?